src/tests/unit/jest.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
         '<rootDir>/src/**/*.{ts,tsx}',
         '!<rootDir>/src/tests/**/*',
         '!<rootDir>/src/**/*.d.ts',
+        '!<rootDir>/src/reports/components/new-tab-link-confirmation-dialog.tsx',
```

Comment:
this seems odd? care to add a comment explaining why we don't want to collect coverage from this? (I have a guess on the reason we need this exception but it would be nice to be a little more explicit)

Reply:
Added `// excluding the following file to omit code cov in generated script for running eval in corresponding test`. Is it sufficient?

Reply:
so... does the use of `eval` messes with the code coverage? If that's what's happening here, I think that should be part of the comment.

Reply:
It's the other way around; the use of code coverage messes with the eval in the test (the product code is functionally evaling the compiled version of the source, which in the jest case includes coverage-injected functions that don't actually exist at runtime)

Reply:
It would be nice to look into whether it's possible to exclude just the specific function from coverage rather than the whole file, though (not sure if there's any sort of special comment syntax for that)

Reply:
```
// excluding the following file to omit code cov in generated script, which would contain
// code cov injected functions that interfere with running eval
```
How's this?

Reply:
According to [this](https://codewithhugo.com/jest-exclude-coverage/), it's possible to ignore a specific line or function